src/Properties.tsx

```diff
@@ -1,65 +1,58 @@
 import React, { Fragment } from 'react';
-import { FeatureCollection } from 'geojson';
-
+import { Feature } from 'geojson';
 interface PropertyProps {
-  info: FeatureCollection[] | null;
+  feature: Feature;
 }
 
-const Properties: React.FC<PropertyProps> = (props: PropertyProps) => {
-  console.log('props', props);
-  const { info } = props;
-  console.log('info', info);
-
-  return (
-    <div>
-      {info ? (
-        info.map((inf, i) =>
-          inf ? (
-            <Fragment key={`${i}-fr-inf`}>
-              <ul key={`${i}-ul-inf`}>
-                Properties:{' '}
-                {inf.features.map((feature, j) => {
-                  const { properties } = feature;
-                  // console.log('properties', properties);
-                  return properties ? (
-                    <Fragment key={`${j}-pr-fr-inf`}>
-                      <br />
-                      <li key={`${properties.fid}-map-properties`}>
-                        {properties.building}
-                        {properties.roomId}
-                      </li>
-                      <li> Floor: {properties.floor}</li>
-                      <li>
-                        {' '}
-                        Room: {properties.roomName} ({properties.roomType})
-                      </li>
-                      <li>
-                        Search Tags:
-                        <ul>
-                          {properties.searchTagProps.map(
-                            (searchTag: string, k: number) => (
-                              <li key={`${k}-search=tag`}>{searchTag}</li>
-                            )
-                          )}
-                        </ul>
-                      </li>
-                      <br />
-                    </Fragment>
-                  ) : (
-                    'N/A'
-                  );
-                })}
-              </ul>
-            </Fragment>
-          ) : (
-            <p key={`${i}-err-inf`}>Something may have gone wrong</p>
-          )
-        )
-      ) : (
-        <p>Open a file!</p>
+const Property = ({ feature }: PropertyProps) => {
+  const { properties } = feature;
+  return properties ? (
+    <Fragment key={`${properties.building}-${properties.roomId}-property`}>
+      <br />
+      <li>
```

Comment:
I'll leave it up to you whether to make this change in this PR, or as a new task, but I think instead of hardcoding the values we display here, we should extract the props from the feature using `Object.entries` and then display the props as simple `key: value` mappings.

Reply:
Facts

I'll update this in the current pr

Reply:
Ok, thinking a bit more about it.

It would be a little hard to do, right? Unless we want every single properties attribute to be wrapped in a different <li> attribute. (Do we want that? This is easily implementable then)

I'd just note that some properties are put in the same <li> element ig.

Reply:
I think we _want_ every prop in a different `<li>`.

Essentially, we want:
```jsx
Object.entries(feature.properties).map( (key, value) => (
  <li key={key}>
    {`${key}: ${value}`}
  </li>
))
```